pliers/extractors/models.py

```diff
@@ -84,3 +84,65 @@ def _extract(self, stim):
             values.append(float(extraction[1]))
 
         return ExtractorResult([values], stim, self, features=features)
+
+
+def _resize_image(image, shape):
+    import numpy as np
+    from PIL import Image
+    return np.array(
+        Image.fromarray(image).resize(shape, resample=Image.BILINEAR))
+
+
+class TensorFlowKerasInceptionV3Extractor(ImageExtractor):
+
+    ''' Labels objects in images using a pretrained Inception V3 architecture
+     implemented in TensorFlow / Keras.
+
+    Args:
+        weights (str): URL to download pre-trained weights. If None (default),
+            uses the pre-trained Inception V3 model (dated 2017-03-10) used in
+            Keras Applications.
+        num_predictions (int): Number of top predicted labels to retain for
+            each image.
+     '''
+
+    _log_attributes = ('weights', 'num_predictions')
+    VERSION = '1.0'
+
+    def __init__(self, weights=None, num_predictions=5):
+        super(TensorFlowKerasInceptionV3Extractor, self).__init__()
+        if weights is None:
+            weights = 'imagenet'
+        self.weights = weights
+        self.num_predictions = num_predictions
+
+        import tensorflow as tf
+        # Instantiating the model also downloads the weights to a cache.
+        self.model = tf.keras.applications.inception_v3.InceptionV3(
```

Comment:
I haven't looked into this in any detail, but if the models here all have the same API, I wonder if we can just replace this with a generic `TensorFlowKerasApplicationExtractor` that takes arbitrary init arguments, and then wrap that with very thin subclasses for the individual extractors? I'm not suggesting you should do this yourself, @kaczmarj, but just wondering whether you think this is a viable approach (in which case we can open an issue for a refactor along these lines).

Reply:
See the [`LibrosaFeatureExtractor`](https://github.com/tyarkoni/pliers/blob/master/pliers/extractors/audio.py#L140) hierarchy for an example of what this might look like.

Reply:
The models do share the same API (go keras!). We probably wouldn't even need subclasses. The differences across models will be their location in the applications package as well as the required input shape. A simple mapping of these things would probably suffice. I can do that too (in a separate PR).